src/static/riot/queues/management.tag

```diff
@@ -186,10 +184,12 @@
         Modal Methods
         ----------------------------------------------------------------------*/
         self.show_modal = (queue) => {
+            // Show modal THEN update!
+            $(self.refs.modal).modal('show')
```

Comment:
Why does this matter? what changes between doing this before vs after?

Reply:
I was trying to get this dropdown to work and part of that was thinking this modal needed to be shown first before the dropdown would refresh properly with the selected organizers.

![image](https://user-images.githubusercontent.com/2185159/73423308-977ed280-42e0-11ea-851f-2ff541eed298.png)


good catch, I tested and this was not part of what made it start working, ty!

✔️